html-forms_personal-profile-form/index.html

```diff
@@ -10,6 +10,34 @@
     <link rel="stylesheet" href="./css/styles.css" />
   </head>
   <body>
-    <section></section>
+      <form>
+        <p>We want to get to know you! Please enter your personal information.</p>
+        <fieldset><h4>Personal information</h4></fieldset>
```

Comment:
i think `fieldset` should contain all these inputs:

 <fieldset>
          <legend>Personal Information</legend>

          <label for="first-name">First Name:</label>
          <input type="text" />

          <label for="last-name">Last Name:</label>
          <input type="text" />

          <label for="birthday">Your Birthday</label>
          <input type="date" />
</fieldset>